recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 
   run:
     - python
```

Comment:
`python =2.7|>=3.6` or `python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,>=2.7` should work, I think